service/routes.py

```diff
@@ -104,3 +104,24 @@ def list_shopcart_items(customer_id):
     )
 
     return jsonify(shopcart_list), status.HTTP_200_OK
+
+
+######################################################################
+# READ AN ITEM FROM A SHOPCART
+######################################################################
+@app.route("/shopcarts/<customer_id>/<item_id>", methods=["GET"])
+def get_item(customer_id, item_id):
+    """
+    Read an item from a shopcart
+    """
+    app.logger.info("Request to read an Item %s from Customer with id: %s 's shopcart", item_id, customer_id)
+
+    # See if the item exists and abort if it doesn't
+    if ShopCarts.check_exist_by_customer_id_and_product_id(customer_id, item_id) == 0:
+         logger.info(f"Customer {customer_id} and corresponding item {item_id} already exists")
```

Comment:
log message should show item doesn't exist and level should probably be error instead of info.